lisa/tools/lsvmbus.py

```diff
@@ -0,0 +1,57 @@
+import re
+from typing import Any, Dict
+
+from lisa.executable import Tool
+from lisa.util import LisaException
+
+from .wget import Wget
+
+
+class Lsvmbus(Tool):
+    __lsvmbus_path = (
+        "https://raw.githubusercontent.com/torvalds/linux/master/tools/hv/lsvmbus"
+    )
+    __matched_str = "Rel_ID"
+
+    @property
+    def command(self) -> str:
+        return self._command
+
+    def can_install(self) -> bool:
+        return True
+
+    def _initialize(self, *args: Any, **kwargs: Any) -> None:
+        self._command = "lsvmbus"
+        self._cached_result: Dict[str, str] = {}
+
+    def install(self) -> bool:
+        wget_tool = self.node.tools[Wget]
+        current_path = self.node.execute("pwd")
+        file_path = wget_tool.get(self.__lsvmbus_path, f"{current_path}/.local/bin")
+        # make the download file executable
+        self.node.execute(f"chmod +x {str(file_path)}")
+        self._command = f"{str(file_path)}"
+        return True
+
+    def get_verbose_messages(self, force_run: bool = False) -> Dict[str, str]:
+        cached_result_dict = self._cached_result
+        cached_result = None
+        if len(cached_result_dict) == 0 or force_run:
+            cached_result = self.run("-vv")
+            if cached_result.exit_code != 0:
+                raise LisaException(
+                    f"get unexpected non-zero exit code {cached_result.exit_code} "
+                    f"when run {self.command} -vv."
+                )
+            raw_array = re.split(r"\n\s*\n", cached_result.stdout)
```

Comment:
Split by lines by `splitlines`. But if to use re, make sure pattern is precompiled, don't compile the string every time.